frontend/public/components/masthead-toolbar.jsx

```diff
@@ -748,6 +758,16 @@ class MastheadToolbarContents_ extends React.Component {
           </ToolbarContent>
         </Toolbar>
         <AboutModal isOpen={showAboutModal} closeAboutModal={this._closeAboutModal} />
+        {this.state.reportBugLink ? (
+          <FeedbackModal
+            onShareFeedback="https://console.redhat.com/self-managed-feedback-form"
+            onReportABug={this.state.reportBugLink}
```

Comment:
For released versions of OpenShift, `reportBugLink` will be the open support case link. Is that what we want? (`FeedbackModal` already has an open support case link, correct?)

Reply:
yeah I was taking that link that has other information you showed me in an message and putting them open a support ticket.  maybe i should rename this prop.  thoughts?

Reply:
I can also leave the report bug link if you want as well...

Reply:
@dlabaj Sorry, I'm still a little unclear what the behavior is. How does the feedback modal look for released vs prereleased versions? And which link does `onReportABug` replace in the modal?

Reply:
The current behavior is:
* OKD: [no link of any kind](https://github.com/openshift/console/blob/d782d58a96464b7b53d5f3501bfe6ea5084ad5c4/frontend/public/module/k8s/cluster-settings.ts#L278-L281)
* OpenShift:
  * prerelease: [open a bug link](https://github.com/openshift/console/blob/d782d58a96464b7b53d5f3501bfe6ea5084ad5c4/frontend/public/module/k8s/cluster-settings.ts#L294)
  * release: [open a support case link](https://github.com/openshift/console/blob/d782d58a96464b7b53d5f3501bfe6ea5084ad5c4/frontend/public/module/k8s/cluster-settings.ts#L287-L289)

Reply:
OK I see. We're passing the object and not just the URL, so the feedback modal uses the same link and label... Is that right?

Reply:
yes that's correct.